.circleci/config.yml

```diff
@@ -1208,6 +1241,10 @@ workflows:
       - t_ubu_asan_constantinople_clang: *workflow_ubuntu2004_asan_clang
       - t_ubu_asan_cli: *workflow_ubuntu2004_asan
 
+      # UBSan build and tests
+      - b_ubu_ubsan_clang: *workflow_trigger_on_tags
+      - t_ubu_ubsan_clang: *workflow_ubuntu2004_ubsan_clang
```

Comment:
CLI run would also benefit. I was a bit cautious because that would introduce another CI and lately we have been debating cost-benefit tradeoff and I wasn't sure where this would tip should CLI have a dedicated UBSan run.

Reply:
I think it's not a big deal for a nightly run. I'd only worry about that if it was a PR check because I suspect most of our usage comes from them.

Reply:
Got it, I will add a UBSAN CLI nightly job in that case.